payments/credit-card/redirect.md

```diff
@@ -597,6 +597,7 @@ next_href="seamless-view" next_title="Next: Seamless View" %}
 [recur]: /payments/credit-card/other-features/#recur
 [redirect-image]: /assets/img/checkout/test-purchase.png
 [reversal]: /payments/credit-card/after-payment/#reversals
-[Screnshot-1]: /assets/img/checkout/test-purchase.png
+[Screenshot-1]: /assets/img/payments/card_payment.png
+{:height="500px" width="425px"}
```

Comment:
Adding image dimensions on the footnote URL doesn't work, they need to be added to the image placement in the document.